clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -10347,6 +10347,8 @@ def err_x86_builtin_tile_arg_duplicate : Error<
 
 def err_builtin_target_unsupported : Error<
   "builtin is not supported on this target">;
+def err_builtin_aix_os_unsupported : Error<
+  "this builtin is available only on AIX 7.2 and later operating systems">;
```

Comment:
```suggestion
  "builtin is available only on AIX 7.2 and later operating systems">;
```

Reply:
I think with `this` grammar correct in the following compiler error output:  (this indicates a certain functions __builtin_cpu_is("power8"); )

builtin.c:2:10: error: this builtin is available only on AIX 7.2 and later operating systems
    2 |   return __builtin_cpu_is("power8");